libs/atqr/domain/src/lib/services/notificationsEnums.ts

```diff
@@ -0,0 +1,40 @@
+export enum NotificationChannel {
+  Email = 'Email',
+  SMS = 'SMS',
+  Push = 'Push',
+}
+
+export enum MessageTemplateId {
+  Challenge = 'Challenge',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeen = 'YourOfficiationRequestHaveBeen',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContender = 'YourContender',
+  TimeToOfficiateArrived = 'TimeToOfficiateArrived',
+  YourJudgeDidNotAnswered = 'YourJudgeDidNotAnswered',
+}
```

Comment:
There are a LOT of missing items here. When you see these in Disl:

```
Challenge...
  ~Created
  ~Canceler
```

You need to create the items:

```
ChallengeCreated = 'ChallengeCreated',
ChallengeCanceled = 'ChallengeCanceled'
```